core/runtime/src/main/java/io/quarkus/runtime/TemplateHtmlBuilder.java

```diff
@@ -12,17 +12,28 @@
             "</head>\n" +
             "<body>\n";
 
-    private static final String HTML_TEMPLATE_END = "</div></body>\n" +
+    private static final String HTML_TEMPLATE_END = "</body>\n" +
             "</html>\n";
 
-    private static final String HEADER_TEMPLATE = "<header>\n" +
-            "    <h1 class=\"container\">%1$s</h1>\n" +
+    private static final String HEADERS_TEMPLATE = "<header>\n" +
+            "    <div class=\"component-name\">\n" +
+            "        <h1 class=\"container\">%1$s</h1>\n" +
+            "    </div>\n" +
             "    <div class=\"exception-message\">\n" +
             "        <h2 class=\"container\">%2$s</h2>\n" +
             "    </div>\n" +
             "</header>\n" +
             "<div class=\"container content\">\n";
 
+    private static final String HEADER_END = "</div>\n";
+
+    private static final String HEADER_TEMPLATE = "<header>\n" +
+            "    <div class=\"exception-message\">\n" +
+            "        <h2 class=\"container\">%1$s</h2>\n" +
+            "    </div>\n" +
+            "</header>\n" +
+            "<div class=\"container content\">\n";
```

Comment:
These changes don't look related to the issue at hand and don't seem like an improvement to me.